packages/build/src/index.ts

```diff
@@ -36,25 +35,24 @@ export async function runBuild(options: BuildOptions) {
     bundle = true,
     format = 'cjs',
     platform = 'node',
-    minify = false,
     external,
     dev = false
   } = options;
 
-  await build({ 
+  await build({
     entryPoints,
     outfile,
     bundle,
-    minify,
+    minify: !dev,
```

Comment:
We now:
- Always minify during production
- Never minify during development

I don't think we were minifying during production before (we were only doing it during dev, oops).

Is this `build()` hook what is also used when generating the production release? If so, this is technically a change in what we'd ship, but probably a good one?